app/liquid_tags/codesandbox_tag.rb

```diff
@@ -48,7 +48,7 @@ def parse_options(input)
   # composed of letters, numbers, dashes, underscores, forward slashes, @ signs, periods/dots,
   # and % symbols.  Invalid options will raise an exception
   def valid_option(option)
-    raise StandardError, "CodeSandbox Error: Invalid options" unless (option =~ /\A(initialpath=([a-zA-Z0-9\-\_\/\.\@\%])+)\Z|\A(module=([a-zA-Z0-9\-\_\/\.\@\%])+)\Z|\A(runonclick=((0|1){1}))\Z/)&.zero?
+    raise StandardError, "CodeSandbox Error: Invalid options" unless (option =~ /\A(initialpath=([a-zA-Z0-9\-\_\/\.\@\%])+)\Z|\A(module=([a-zA-Z0-9\-\_\/\.\@\%])+)\Z|\A(runonclick=((0|1){1}))\Z|\Aview=(editor|split|preview)\Z/)&.zero?
```

Comment:
could we move this giant regexp to a constant up in the class?

Something like:

```ruby
OPTIONS_REGEXP = /stuff/.freeze
```

Reply:
It's my first Ruby code I've ever written, but I can definetely do that! :)

Reply:
wow @erykpiast, thanks a ton then! Don't worry if you find it tricky, the PR is working already (that's why I accepted it).

Welcome to DEV and welcome to Ruby ;-)